alpine/Dockerfile

```
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/alpine/alpine.tmpl.dockerfile.
 FROM postgres:13.4-alpine
```

I think it is time we bump the alpine base image to PG 14

```suggestion
FROM postgres:14.0-alpine
```